### ISSUE TITLE
How can i plot validation loss and training loss on tensorboard?

### ISSUE THREAD

**Issue by bushra-hafeez** [AntonMu/TrainYourOwnYOLO#169]
Hi,
I had succeeded in training the model on my own dataset, I had 6k training images of which i performed 1k+ annotations and 2k test images... The tensorboard results can be seen in screenshot attached. 

My question is that is there any way to plot validation loss along with training loss? So that i could compare them. Also, the training loss graph seems to have no spikes (too smooth) in its graph, is that good or is my model overfitting?
![Screenshot_20200827_162720](https://user-images.githubusercontent.com/60045644/91438515-ed225a00-e884-11ea-89de-5a02c60439da.jpg)

**Comment by AntonMu** [comment]
Hi @bushra-hafeez,

Your loss plot looks good to me. To plot them in the same graph, you would have to add a new callback that plots e.g. the difference of val loss and loss. 

An easier solution is to just download the raw data for the validation loss and training loss from tensorboard and then put them in an excel file or a jupyter notebook. 

Do be able to download the csv from tensorboard, you need to enable the `Show data download links` option. 

![chrome_nM4flJInGY](https://user-images.githubusercontent.com/17406602/91652238-01638400-ea4a-11ea-8fae-363e47e0bcfc.png)

Tensorboard makes separate plots for each experiment parameter/metric but across multiple runs. For instance, if you train 20 models, then there would be 20 different loss curves all plotted in the same chart. Now if you were to add also another metric in the same graph it would get very confusing. 

Hope this helps.

**Comment by AntonMu** [comment]
Hi @bushra-hafeez,

I ported the repo to TensorFlow 2.3 over the weekend and with the latest release of tensorboard, the loss curves are plotted in the same chart. 

![chrome_XN3v48YnuJ](https://user-images.githubusercontent.com/17406602/91800907-bdee4e80-ebde-11ea-956c-b1e412d12576.png)

Enjoy!
